rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -351,9 +419,33 @@ void RingBuffer::increment_write_ptr()
     _wait_for_load.notify_all();
 }
 
-void RingBuffer::set_meta_data( ImageNameBatch names, pMetaDataBatch meta_data)
+void RingBuffer::set_meta_data(ImageNameBatch names, pMetaDataBatch meta_data, bool is_segmentation)
```

Comment:
is_segmentation is not used here, do we need to pass?

Reply:
Done